docs/source/api/v4/steering_id_targets_targetID.rst

```diff
@@ -105,6 +106,7 @@ Removes a specific target mapping from a specific :term:`Delivery Service`
 
 :Auth. Required: Yes
 :Roles Required: Portal, Steering, Federation, "operations" or "admin"
+:Permissions Required: STEERING:DELETE, STEERING:READ, DELIVERY-SERVICE:READ, TYPE:READ, DELIVERY-SERVICE:UPDATE
```

Comment:
I don't think we should require `DELIVERY-SERVICE:UPDATE` here, just because there is a desire for users without the ability to directly manage Delivery Services to be able to manipulate steering assignments, which would be made impossible by requiring that Permission.